### PR TITLE
Sb 14353

### PR DESCRIPTION
**Pull request by manzarul**
* Changes in how to docs
* user creation
* org creation
* bulk user creation
* user-token 
* sso configuration

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://481.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_bulk_user_create/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_org_add_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_organization/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user_auth_token/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/sso_config_learner-service/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://481.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_bulk_user_create/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_org_add_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_organization/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user_auth_token/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/sso_config_learner-service/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://481.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_bulk_user_create/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_org_add_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_organization/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user_auth_token/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/sso_config_learner-service/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://481.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_bulk_user_create/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_org_add_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_organization/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user_auth_token/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/sso_config_learner-service/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://481.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_bulk_user_create/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_org_add_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_organization/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user_auth_token/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/sso_config_learner-service/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://481.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_bulk_user_create/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_org_add_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_organization/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/how_to_create_user_auth_token/ <br /> - http://481.qa.docs.sunbird.org/2.0.0/developer-docs/how-to-guide/sso_config_learner-service/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/how-to-guide/how_to_bulk_user_create.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+2. API Key to access Sunbird APIs. To create an API key refer [How to generate a Sunbird API key](developer-docs/how-to-guide/generate_apikey/)
+  
+3. Software that can make API calls like curl or [POSTMAN](https://www.getpostman.com/docs/v6/postman/api_documentation/intro_to_api_documentation)
+4. Generating user token [How to generate x-authenticated-user-token] (developer-docs/how-to-guide/how_to_create_user_auth_token/)
+5. Access to [User API](apis/bulkupload/#operation/bulk%20upload%20user%20req1)
+6. A text file that has all user's data in .csv format.
+
+### Taskflow 
+User bulk upload, User can be uploaded under under rootOrg at a time. User rootOrg will be define based on uploader x-authenticated-user-token.
```
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply developer-docs/how-to-guide/how_to_bulk_user_create.md]: Comments not applicable as feature is deleted

File: developer-docs/how-to-guide/how_to_bulk_user_create.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ 
+
+**CSV headers**
+
+|     column    |          Type  | Description | Sample |
+|---------------|----------------|-----------|--------|
+| NAME          | String         | Name of the user  | Mr John |
+|MOBILE PHONE   | String         | The user's mobile phone number | 9012333300|
+|Email          | String         | The user's email address. It is mandatory to specify either the user's mobile phone number or email address| test@gmail.com |
+|SCHOOL ID      | String         | The organisation's unique identifier  | 12344444
```
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply developer-docs/how-to-guide/how_to_bulk_user_create.md]: Comments not applicable as feature is deleted

File: developer-docs/how-to-guide/how_to_bulk_user_create.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+|USER_TYPE      | String         | The type of user as per the organization  | Currently, the values supported are 'teacher' and 'other'
+|ROLES          | list           | A list of roles supported in the orgnaization | role1, role2
```
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply developer-docs/how-to-guide/how_to_bulk_user_create.md]: Comments not applicable as feature is deleted

File: developer-docs/how-to-guide/how_to_bulk_user_create.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+|USER ID        | String         | The unique identifier for a user. This is mandatory if you want to update user details| 
+|External ID    | String         | either school id or external id is mandatory to associate user with org
```
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply developer-docs/how-to-guide/how_to_bulk_user_create.md]: Comments not applicable as feature is deleted

File: developer-docs/how-to-guide/how_to_bulk_user_create.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+|USER ID        | String         | The unique identifier for a user. This is mandatory if you want to update user details| 
+|External ID    | String         | either school id or external id is mandatory to associate user with org 
+
+
+
+**Request Header**
+
+|     Header    |          Type         | Description | Sample |
+|---------------|------------------------|-----------|--------|
+| Content-type | String | Mime type of the request | multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW |
```
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply developer-docs/how-to-guide/how_to_bulk_user_create.md]: Comments not applicable as feature is deleted

**Pull request merged**